prow/cmd/branchprotector/README.md

```diff
@@ -0,0 +1,168 @@
+
+branchprotector configures [github branch protection] according to a specified
+policy.
+
+## Policy configuration
+
+Extend the primary prow [`config.yaml`] document to include a top-level
+`branch-protection` key that looks like the following:
+
+```yaml
+
+branch-protection:
+  orgs:
+    kubernetes:
+      repos:
+        test-infra:
+          # Protect all branches in kubernetes/test-infra
+          protect-by-default: true
+          # Always allow the org's oncall-team to push
+          pushers: ["oncall-team"]
+          # Ensure that the extra-process-followed github context passes
+          # In addition, adds any always_run: True prow jobs
+          require-contexts: ["extra-process-followed"]
```

Comment:
The status checks, as [GitHub calls them](https://help.github.com/articles/about-required-status-checks/), are called contexts here. Looking at the test infra docs I didn't see where contexts is defined. Is it more than the status checks? If not, whey are we renaming them to something else?

Reply:
Done